packages/commercetools/theme/components/TopMenu.vue

```diff
@@ -0,0 +1,161 @@
+<template>
```

Comment:
How about calling the file `HeaderNav` and putting it in `components/header` along with `AppHeader` component? If these components are not global and always used together it is easier to maintain them if they're in the same directory